ci/test/00_setup_env_s390x_host.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+export HOST=s390x-linux-gnu
+export RUN_CI_ON_HOST=true
```

Comment:
This is a dangerous setting. Would be best to put this in the .travis.yml to make sure no one is setting this locally and trashing their machine?

Reply:
> The setting should probably be renamed to DANGER_RUN_CI_ON_HOST or so

Implemented.